src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "5043CECEE6A748679CBE02D27D83747A",
+  "ActionKeyword": "$",
+  "Name": "Windows Settings",
+  "Author": "Microsoft",
```

Comment:
Any reason why this plugins is authored by Microsoft whereas the registry plugin is authored by TobiasSekan?

Reply:
Ouh 😄 I didn't have seen this